css/style.css

```diff
@@ -1486,36 +1485,40 @@ a.fa-linkedin {
 .slider:before {
   position: absolute;
   content: "";
-  height: 15px;
-  width: 15px;
-  left: 3px;
-  bottom: 1.5px;
-  background-color: white;
+  height: 18px;
+  width: 18px;
+  bottom: -.8px;
+  background: white url("../assets/Images/mode/sunny.png");
```

Comment:
as I have mentioned before, please name the  sub-directories  inside the assets directory judiciously so that it becomes easy for us to document this. Please make it ```dark_mode_switch/sunny.png```